frontend/src/tests/instructor-tests.js

```diff
@@ -9,7 +9,8 @@ import {
     expect,
     it,
     beforeAll,
-    afterAll
+    afterAll,
```

Comment:
Where is this used? It probably shouldn't be...

Reply:
I didn't change this

Reply:
Please look into it and file an issue/PR if needed.